images/runtime/node/Dockerfile.template

```diff
@@ -22,6 +22,13 @@ ENV ORYX_AI_INSTRUMENTATION_KEY=${AI_KEY}
 RUN mkdir -p /node_modules \
  && chmod 777 /node_modules
 
+RUN set -e \
+ && npm_ver=$(npm --version) \
+ && if [ ! "$npm_ver" = "${npm_ver#6.}" ]; then \
+        echo "Upgrading npm version from $npm_ver to 6.9.0"; \
+        npm install -g npm@6.9.0; \
```

Comment:
what about running this command but passing a version spec that is the latest version of the major version?
e.g. `npm install -g npm@${npm:0:1}`?

Reply:
if we use globally I think it gets store in /usr/locl/bin something like that? Will that not cause problem as  our links  are in opt?

Reply:
yeah i thought about it but felt it is too broad of a change to do now...what do you think?

Reply:
> if we use globally I think it gets store in /usr/locl/bin something like that? Will that not cause problem as our links are in opt?

This is runtime image and not build image and the global install puts it in where the original npm in installed, so we are good